packages/core/database/src/utils/identifiers/shortener.ts

```diff
@@ -134,14 +135,16 @@ export function getShortenedName(name: string, len: number) {
  * compressed sufficiently, an error is thrown. This function supports dynamic adjustment of token lengths to fit within the
  * maxLength constraint (that is, it will always make use of all available space), while also ensuring the preservation of
  * incompressible tokens.
- *
  * @param {NameToken[]} nameTokens - Array of name tokens
- * @param {number} [maxLength] - Maximum length for the final name string.
+ * @param {NameFromTokenOptions} options - Name token options
  * @returns {string} The generated name string within maxLength.
```

Comment:
I don't think we need to keep those jsdoc tags since we already have the TS types